src/main/java/com/endava/soa_5/basic/Exercise_2.java

```diff
@@ -0,0 +1,32 @@
+package com.endava.soa_5.basic;
+
+public class Exercise_2 {
+    //Write a method that prints if a number is divisible by 3 or 5.
+
+    int number;
+
+    Exercise_2() {
```

Comment:
add a default value of number parameter, otherwise it will be 0

Reply:
Done!